### ISSUE TITLE
Giving less accuracy while running predict.py

### ISSUE THREAD

**Issue by himabinduyeddala** [davidsandberg/facenet#522]
Hi davidsandberg,
                      while running predict.py i got less accuracy like this --
![1_op](https://user-images.githubusercontent.com/30427692/32597169-d799f264-c55b-11e7-882c-d2ecab57b4e8.png)
please resolve my issue why its given less accuracy as early as possible.

Thanks in Advance

**Comment by p0werHu** [comment]
Hi,using predict.py needs classifier.pkl.But I don’t konw how to get this file. Could you told me how to get it? Thank you.

**Comment by himabinduyeddala** [comment]
Hi@Tom Hu,
              First you need to create .pkl file for your dataset, for that run classifer.py in train mode with your datase thent you will get pkl file.

python src/classifier.py TRAIN /home/david/datasets/lfw/lfw_mtcnnalign_160 /home/david/models/model-20170216-091149.pb ~/models/lfw_classifier.pkl --batch_size 1000 --min_nrof_images_per_class 40 --nrof_train_images_per_class 35 --use_split_dataset

Thanks@Regards
Bindu

**Comment by moonyoung430** [comment]
@himabinduyeddala  Hi, How do you get the file 20170216-091149.pb and lfw_classifier.pkl??

**Comment by himabinduyeddala** [comment]
Hi @moonyoung430 ,
   For creating .pb file you need to run freezegraph.py &
for creating .pkl file you need to run classifier.py with mode= TRAIN.

Best
Himabindu

**Comment by moonyoung430** [comment]
@himabinduyeddala Hi,I run classifer.py and created .pkl, downloaded the pretrain model 20170512--110547,then run predict.py .  
I met the same problem with you, and I guess the problem is classifer,I will check classifier.py
